packages/simulator/web/routes.js

```diff
@@ -71,16 +71,14 @@ function register(io) {
     engine.postombud.pipe(toArray()).subscribe((postombud) => {
       socket.emit('postombud', postombud)
     })
-    engine.busStops.subscribe(busStops =>
-      socket.emit('busStops', busStops)
-    )
+    engine.busStops.subscribe((busStops) => socket.emit('busStops', busStops))
 
     engine.kommuner
-      .pipe(map(({id, name, geometry}) => ({id, name, geometry})))
+      .pipe(map(({ id, name, geometry }) => ({ id, name, geometry })))
       .subscribe((kommun) => socket.emit('kommun', kommun))
 
     merge(engine.dispatchedBookings, engine.busStopTimes)
-      .pipe(cleanBookings(), bufferTime(100, null, 1000))
```

Comment:
This was removed because the booking from bus were already "clean". Does this cause some side effects to the previous scenarios with bookings/transports?